src/app/leaves/add/components/LeaveRequestFormBuilder.tsx

```diff
@@ -57,15 +60,24 @@ const LEAVE_REQUEST_TYPES = [
   LeaveTypes.Vacation,
 ];
 
-const user = {
-  id: 3232,
-  role: UserRole.Member,
+const loginUser: User = {
+  id: 1,
+  name: "Alice Johnson",
+  shift: "A",
+  role: "incharge",
+  status: UserStatus.Active,
+  phone: "123-456-7890",
+  address: "123 Main St, Anytown, USA",
+  created_at: new Date().toISOString(),
+  updated_at: new Date().toISOString(),
+  deleted_at: null,
 };
+
 export const LeaveRequestFormBuilder = () => {
   const form = useForm<z.infer<typeof leaveRequestSchema>>({
     resolver: zodResolver(leaveRequestSchema),
     defaultValues: {
-      memberId: user.role === UserRole.Member ? user.id.toString() : "",
+      memberId: loginUser.role === UserRole.Member ? loginUser.id.toString() : "",
```

Comment:
@nadirhussain786 we need to learn how to not get our PRs out of context. This PR was only for form title but we expanded to other areas.

Reply:
Understood, Sir. I'll focus on these areas next time.